octobot_trading/exchanges/util/parser/util.py

```diff
@@ -0,0 +1,291 @@
+import decimal
+from octobot_trading import constants as trading_constants
+from octobot_trading.enums import (
+    ExchangePositionCCXTColumns,
+)
+
+
+class Parser:
```

Comment:
I don't think parsers should be placed in "util" package. I think it should have its own package

Reply:
its now in exchange/parser